be/btree.c

```diff
@@ -10280,14 +10285,14 @@ static void btree_ut_kv_oper_thread_handler(struct btree_ut_thread_info *ti)
 	 */
 	M0_ASSERT(ti->ti_key_size == RANDOM_KEY_SIZE ||
 		  (ti->ti_key_size % sizeof(uint64_t) == 0 &&
-		   ti->ti_key_size > sizeof(key[0]) &&
+		   ti->ti_key_size >= sizeof(key[0]) &&
 		   ti->ti_key_size <= sizeof(key)));
 	M0_ASSERT(ti->ti_value_size == RANDOM_VALUE_SIZE ||
-		  (ti->ti_value_size % sizeof(uint64_t) == 0 &&
+		  (ti->ti_value_size % sizeof(uint32_t) == 0 &&
```

Comment:
Please add comment here something like:
`Here **ti->ti_value_size % sizeof(uint32_t)** check is added to handle the cases where value size is not in multiple of 8 bytes.`
Also mention the particular structure and ut name in above comment due to which this special handling is needed.

Reply:
Done